oak_functions/proto/server.proto

```diff
@@ -33,19 +38,17 @@ message Request {
 
 message AttestedInvokeRequest {
   oneof request_type {
-    // Initialization message that starts the Remote Attestation protocol.
-    oak.remote_attestation.ClientHello client_hello = 1;
-    // Client part of session key negotiation.
-    oak.remote_attestation.ClientIdentity client_identity = 2;
+    // Remote Attestation handshake message.
+    AttestationMessage attestation_message = 1;
     // Request with payload encrypted using the session key.
-    Request request = 3;
+    Request request = 2;
```

Comment:
Why is there still this message here? I was expecting that the service definition becomes effectively bytes in, bytes out; everything else is just about how those bytes are serialized and deserialized. 

Also I thought that the serialization is now done manually (I saw bincode being used), so why are all the types still defined as protobuf too?

Reply:
Do we want to have bytes everywhere? I thought we wanted to have bytes in the general remote attestation protocol, but still have types in the Oak Functions server.

Reply:
I added it because it's a `oneof` Protobuf field. And having `bytes .. = 1` and `bytes .. = 2` is impossible.

Reply:
I'm curious why is it impossible? I thought they are just separate fields.

Reply:
IIUC it checks the type of a `oneof` variant, not it's name.

Reply:
It could be using bytes, but when there will be no `oneof`. Just `bytes` that are passed to the attestation engine.